CHANGELOG.md

```diff
@@ -1,3 +1,79 @@
+### citus v9.0.0 (October 7, 2019) ###
+
+* Adds support for PostgreSQL 12
+
+* Adds UDFs to help with PostgreSQL upgrades
+
+* Delegates `SELECT function()` to a worker node
+
+* Distributes types to worker nodes
+
+* Introduces `create_distributed_function` UDF
+
+* Introduces local query execution for Citus MX
+
+* Implements infrastructure for routing `CALL` to MX workers
```

Comment:
Maybe move `Delegates `SELECT function()` to a worker node` to  here, with the same wording?

Reply:
Put an item here

`Implements infratructure for routing `SELECT function()` to MX workers`

and removed

`Delegates SELECT function() to a worker node`